### PR TITLE
DAOS-6965 Test: Increased the SCM size as recent layout change need more space with current dataset.

### PR DESCRIPTION
**Pull request by ravalsam**
The test is failing because of SCM size target size is getting low 9MB with the dataset.
This test has no requirement to use low size so increase enough so the test should not fail with ENOSPACE.
Updated test to use PCMEM instead of using tmpfs.

Test-tag-hw-large: nvme_object_multiple_pools

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

**Comment by ravalsam** [comment]
The test is passed in Ci https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-4928/4/artifact/Functional_Hardware_Large/nvme/nvme_object.py/job.log

File: src/tests/ftest/nvme/nvme_object.py [hunk 1]
```diff
@@ -194,6 +195,7 @@ def test_nvme_object_multiple_pools(self):
         threads = []
         index = 0
         for size in self.pool_size[:-1]:
+            time.sleep(1)
```
Comment (rpadma2) [inline src/tests/ftest/nvme/nvme_object.py:198]: Are we sure 1 second is enough? Why we added this 1 second delay? Hope we will not land in some intermittent failure.

**Review by saurabhtandan — APPROVED** [review]
LGTM

File: src/tests/ftest/nvme/nvme_object.py (hunk 1 above)
Comment (ravalsam) [reply src/tests/ftest/nvme/nvme_object.py:198]: Added so threads have some time in between before it begins.

**Review by dinghwah — APPROVED** [review]
LGTM.

**Review by rpadma2 — APPROVED** [review]

**Review by sylviachanoiyee — CHANGES_REQUESTED** [review]
I don't see the nvme test result in Jenkins in build #10. Was the test run?

**Comment by ravalsam** [comment]
> I don't see the nvme test result in Jenkins in build #10. Was the test run?

Yes it was run in #4 build and the result can be found from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-4928/4/artifact/Functional_Hardware_Large/nvme/nvme_object.py/job.log 
I rebased a couple of times because of Ci issues so it's wont be present in latest run

**Pull request merged**